readme.md

```diff
@@ -1,17 +1,31 @@
 # Construct
 
+Construct helps you to quickly setup your computer with the tools we have found helpful here at Segment.
 
-Install construct
+Construct will install the following tools:  
 
-  git clone git@github.com:pivotal-cloudops/construct.git
+  atom
```

Comment:
@supertopher makes sense.  I think I would opt for somehow automatically updating the list so that I can be even more explicit for those that are not super familiar with github or file structures.